### ISSUE TITLE
Innaccuracy: Remove or seriously rework the checklist.

### ISSUE THREAD

**Issue by yatil** [a11yproject/a11yproject.com#132]
I am deeply worried about the checklist part of the web site. It leaves the impression that this is everything that is to do to make a website accessible.

The whole list oversimplifies accessibility issues, as it only shows a small bit of what accessible web design is about. It is not just a technical issue.

Some of the tips may even be harmful, so is unobtrusive JavaScript a good sign for an accessible web site, but if you register a click event on a span for example it won’t be accessible by keyboard, no matter if you use inline JavaScript or unobtrusive techniques.

In the “Forms” section the site speaks about “logical layout” which isn’t testable and imprecise. The WCAG2 has very specific and testable criteria what a good form is and how it should work. To call that a “logical layout” may be okay for accessibility specialists (as they have the accessibility background), but says nothing to the beginner. 

The title on the search field isn’t sufficient either. I’d recommend to use a hidden label and of course a role of search on the parent form. Speaking of roles: The ARIA roles come almost without any comments and should be better described. For example it should be clear what can be used once on a site and what’s better suited for multiple usage.

You recommend transcripts for audios, but not subtitles for videos.

In the color section I’d recommend a better description and talking about contrast more, especially underlining links and the importance of the :focus state.

Having a good document outline is also a must.

“Test using a screen reader” isn’t really what people should do if they haven’t much knowledge of the inner workings. I know using ChromeVox can help understand what’s going on, but working with a real screen reader is too much, especially as they are highly configured for the needs of the individual person.

I think it would really help to point out the grand scheme of things, mention content strategic topics (when we analyse sites, bad navigation structures are one of the major issues) and link to the relevant WCAG techniques.

I hope this helps to improve the project. I will try to contribute some pull requests myself, but in general I’m opposed on the checklist thinking in terms of web accessibility.
